dotnet/src/SemanticKernel.Abstractions/Http/HttpHeaderConstant.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+
+namespace Microsoft.SemanticKernel.Http;
+
+/// <summary>
+/// Provides HTTP header names and values for common purposes.
+/// </summary>
+public static class HttpHeaderConstant
```

Comment:
HttpHeaderConstant is probably not the best container for the user agent setting, since it may not be a header in all cases.  I wouldn't necessarily expect this to be the location for this setting.  Maybe we should consider a new class such as `SemanticKernelSettings` or `DefaultSemanticKernelSettings`